performance/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# syntax=docker/dockerfile:1
+ARG NODE_VERSION=16
+ARG K6_VERSION=0.46.0
+
+FROM node:${NODE_VERSION}-alpine as builder
+
+WORKDIR /build
+
+# Copy over all source code
+COPY . .
+
+# Install build dependencies
+RUN npm install
+
+# Run build scripts for test scripts
+RUN npm start
+
+FROM grafana/k6:${K6_VERSION} as base
```

Comment:
Doing it this way means Dependabot can't automatically open update PRs for us, and we won't get notified of security issues. Do we anticipate needing to run this with different container versions often enough that editing the Dockerfile is more painful than manually keeping track of both upstream base images?

Reply:
I figure letting dependabot bump automatically is gonna yield better results than making the version easier for humans to bump. I'll switch em over.